Whiskey/Tasks/Pester4.ps1

```diff
@@ -30,33 +30,85 @@ function Invoke-WhiskeyPester4Task
     if( $Exclude )
     {
         $Exclude = $Exclude | Convert-WhiskeyPathDirectorySeparator 
-        $path = $path |
-                    Where-Object {
-                        foreach( $exclusion in $Exclude )
+    }
+
+    if( -not $Script )
+    {
+        Stop-WhiskeyTask -TaskContext $TaskContext -PropertyName 'Script' -Message ('Script is mandatory.')
+        return
+    }
+
+    $Script = & {
+        foreach( $scriptItem in $Script )
+        {
+            $path = $null
+
+            if( $scriptItem -is [String] )
+            {
+                $path = $scriptItem
+            }
+            elseif( $scriptItem | Get-Member -Name 'Keys' )
+            {
+                $path = $scriptItem['Path']
+                $numPaths = ($path | Measure-Object).Count
+                if( $numPaths -gt 1 )
+                {
+                    Stop-WhiskeyTask -TaskContext $TaskContext -PropertyName 'Script' -Message ('when passing a hashtable to Pester''s "Script" parameter, the "Path" value must be a single string. We got {0} strings: {1}' -f $numPaths,($path -join ', '))
+                    continue
+                }
+            }
+
+            $path = Resolve-WhiskeyTaskPath -TaskContext $TaskContext -PropertyName 'Script' -Path $path -Mandatory
+
+            foreach( $pathItem in $path )
+            {
+                if( $Exclude )
+                {
+                    $excluded = $false
+                    foreach( $exclusion in $Exclude )
+                    {
+                        if( $pathItem -like $exclusion )
                         {
-                            if( $_ -like $exclusion )
-                            {
-                                Write-WhiskeyVerbose -Context $TaskContext -Message ('EXCLUDE  {0} -like    {1}' -f $_,$exclusion)
-                                return $false
-                            }
-                            else
-                            {
-                                Write-WhiskeyVerbose -Context $TaskContext -Message ('         {0} -notlike {1}' -f $_,$exclusion)
-                            }
+                            Write-WhiskeyVerbose -Context $TaskContext -Message ('EXCLUDE  {0} -like    {1}' -f $pathItem,$exclusion)
+                            $excluded = $true
+                        }
+                        else
+                        {
+                            Write-WhiskeyVerbose -Context $TaskContext -Message ('         {0} -notlike {1}' -f $pathItem,$exclusion)

```

Comment:
This verbose message is a little confusing. I'm going to be running a build with `-Verbose` and then see a message like:
```
VERBOSE:          something.tests.ps1 -notlike dontrunme.tests.ps1
```
I'll know that statement is true but not what is means or why it's showing me this. What is being conveyed by showing a user this statement (other than it's `true`)? This reads more like a DEBUG message.

Reply:
Debug messages are for developers to see to diagnose problems. Verbose messages are for the user to help diagnose problems. If someone's wildcard expressions aren't working, these messages are helpful to a user to figure what is being matched against what so they can get their wildcards correct.